.travis.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     - &test
       stage: test
       name: "Test for 2.12"
-      script: sbt ++$TRAVIS_SCALA_VERSION coverage test coverageReport package packageSrc
+      script: sbt ++$TRAVIS_SCALA_VERSION coverage test coverageReport package packageSrc publishLocal
```

Comment:
What does this give us?

Reply:
publishing too needs to be CIed (see https://github.com/scalaz/scalaz-zio/pull/404)